tests/AutoReview/DocumentationTest.php

```diff
@@ -189,6 +190,39 @@ public function testListingDocumentationIsUpToDate(): void
         );
     }
 
+    public function testAllReportFormatsAreInUsageDoc(): void
+    {
+        $locator = new DocumentationLocator();
+        $usage = $locator->getUsageFilePath();
+        \assert(is_file($usage));
```

Comment:
why native `assert` and not PHPUnit assertion?

Reply:
This test isn't about if the file exists or not (that would indeed be a unitt->assert) but about something else.
The file however must exist in order to be able to perform the test, so its existence is not part of the test but a requirement of the test itself.

Think of it like, if you have to assert/test a car can drive 100 miles, than you unit assert that it drove 100 miles or more at the end of the test.
You might assert before you perform the that you have at least 100 miles or road to test on. You do not unittest there is enough road to test on, as the road length is a requirement to perform the test, you are not testing the length of the road.
If there is not enough road to test on, than the test itself failed, but does not prove the car cannot drive 100 miles.

Reply:
also happy to change to utest->assert if that is what is wanted

Reply:
>maybe we should migrate to native assert everything ;)

or do it the other way around, migrate any `\asset(X)` in a utest to `self::assertTrue(X)`, freeing all from TDD principles, designing your tests as you would code and just skyrocket the assert count ;)